pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -207,14 +207,19 @@ func getStableKey(pod *v1.Pod, container *v1.Container) string {
 
 // buildContainerLogsPath builds log path for container relative to pod logs directory.
 func buildContainerLogsPath(containerName string, restartCount int) string {
-	return fmt.Sprintf("%s_%d.log", containerName, restartCount)
+	return fmt.Sprintf("%s/%s_%d.log", containerName, containerName, restartCount)
```

Comment:
I feel like we can use `restartCount` as log file name for now until we have more metadata to add. Just to make the path shorter.
@mrunalp @feiskyer WDYT?

Reply:
Done